test/Nancy.Testing.Tests/QueryWrapperTests.cs

```diff
@@ -35,12 +40,11 @@ public void Should_filter_selection_when_second_indexer_invoked()
         {
             // Given
             var document =
-                CQ.Create(
-                    @"<html><head></head><body><table><tr><td></td><td></td></tr><tr><td></td><td></td></tr></body></html>");
+                this.parser.Parse(@"<html><head></head><body><table><tr><td></td><td></td></tr><tr><td></td><td></td></tr></body></html>");
 
-            var wrapper = new QueryWrapper(document);
+            var wrapper = new QueryWrapper(new[] { document.DocumentElement });
 
-            var row = wrapper["tr:first"];
+            var row = wrapper["tr:first-child"];

```

Comment:
infact, after running under jsfiddle, it looks like this shouldn't've ever worked, :first doesn't return anything, but :first-child does.[jsfiddle test](https://jsfiddle.net/hjac6Lyn/)